### PR TITLE
Update sql-server-linux-shared-disk-cluster-red-hat-7-configure.md

### PR DESCRIPTION
**Pull request by tanaka-takayoshi**
I got the error by executing the original command because the pcs doesn't have ``defaults`` operation. If we want to change the default timeout, we can execute like ``pcs resource op defaults timeout=60s``. This is valid for all operations, so I put this command at the last line.
Could you check the original command doesn't work and change the document?

### CODE REVIEW

**Comment by PRMerger3** [comment]
@tanaka-takayoshi : Thanks for your contribution to the SQL documentation! The author, @MikeRayMSFT, has been notified to review your proposed change.

**Comment by MikeRayMSFT** [comment]
@tanaka-takayoshi - thank you for your submission. We are reviewing the submitted change.

**Comment by arsing** [comment]
We definitely don't need or want people to change their cluster defaults, so `pcs resource op defaults` is not correct.

I'm not sure why the `defaults` was there in the first place. Just remove it entirely.

File: docs/linux/sql-server-linux-shared-disk-cluster-red-hat-7-configure.md [hunk 1]
```diff
@@ -342,24 +342,26 @@ At this point both instances of SQL Server are configured to run with the databa
...
    sudo pcs cluster cib cfg
-   sudo pcs -f cfg resource create mssqlha ocf:mssql:fci op defaults timeout=60s
-   sudo pcs -f cfg resource create virtualip ocf:heartbeat:IPaddr2 ip=10.0.0.99
+   sudo pcs -f cfg resource create mssqlha ocf:mssql:fci
+   sudo pcs -f cfg resource create virtualip ocf:heartbeat:IPaddr2 ip=10.0.0.99
    sudo pcs -f cfg resource create fs Filesystem device="10.8.8.0:/mnt/nfs" directory="/var/opt/mssql/data" fstype="nfs"
    sudo pcs -f cfg constraint colocation add virtualip mssqlha
    sudo pcs -f cfg constraint colocation add fs mssqlha
    sudo pcs cluster cib-push cfg
+   sudo pcs resource op defaults timeout=60s
```
Comment (MikeRayMSFT) [inline docs/linux/sql-server-linux-shared-disk-cluster-red-hat-7-configure.md]: Line 364 should be removed completely. No default set here.

File: docs/linux/sql-server-linux-shared-disk-cluster-red-hat-7-configure.md [hunk 2]
```diff
@@ -342,24 +342,26 @@ At this point both instances of SQL Server are configured to run with the databa
...
    ```bash
    sudo pcs cluster cib cfg 
-   sudo pcs -f cfg resource create <sqlServerResourceName> ocf:mssql:fci op defaults timeout=<timeout_in_seconds>
+   sudo pcs -f cfg resource create <sqlServerResourceName> ocf:mssql:fci
    sudo pcs -f cfg resource create <floatingIPResourceName> ocf:heartbeat:IPaddr2 ip=<ip Address>
    sudo pcs -f cfg resource create <fileShareResourceName> Filesystem device=<networkPath> directory=<localPath>         fstype=<fileShareType>
    sudo pcs -f cfg constraint colocation add <virtualIPResourceName> <sqlResourceName>
    sudo pcs -f cfg constraint colocation add <fileShareResourceName> <sqlResourceName> 
    sudo pcs cluster cib-push cfg
+   sudo pcs op defaults timeout=<timeout_in_seconds>
```
Comment (MikeRayMSFT) [inline docs/linux/sql-server-linux-shared-disk-cluster-red-hat-7-configure.md]: Line 351 should be removed completely. No default here.

File: docs/linux/sql-server-linux-shared-disk-cluster-red-hat-7-configure.md [hunk 3]
```diff
@@ -354,8 +354,8 @@ At this point both instances of SQL Server are configured to run with the databa
 
    ```bash
    sudo pcs cluster cib cfg
-   sudo pcs -f cfg resource create mssqlha ocf:mssql:fci op defaults timeout=60s
-   sudo pcs -f cfg resource create virtualip ocf:heartbeat:IPaddr2 ip=10.0.0.99
+   sudo pcs -f cfg resource create mssqlha ocf:mssql:fci
+   sudo pcs -f cfg resource create virtualip ocf:heartbeat:IPaddr2 ip=10.0.0.99
```
Comment (Arnavion) [inline docs/linux/sql-server-linux-shared-disk-cluster-red-hat-7-configure.md]: Use normal space, not nbsp.

Comment (arsing) [inline docs/linux/sql-server-linux-shared-disk-cluster-red-hat-7-configure.md]: Use normal space, not nbsp.

**Comment by arsing** [comment]
>- **Timeout Value**: The timeout value is the amount of time that the cluster waits while a a resource is brought online. For SQL Server, this is the time that you expect SQL Server to take to bring the `master` database online.

Remove this too (line 333). After that, LGTM.

**Comment by MikeRayMSFT** [comment]
#sign-off

**Pull request merged**